src/stylesheets/elements/_buttons.scss

```diff
@@ -1,19 +1,19 @@
 .usa-button {
   @include u-margin-y("1px");
-  @include u-text('secondary-darker');
+  @include u-text("secondary-darker");
   @include u-padding-y(1.5);
   @include u-padding-x(4);
 
   &.usa-button--hover,
   &:hover {
-    @include u-bg('primary-light');
-    @include u-text('secondary-darker');
+    @include u-bg("primary-light");
```

Comment:
Remove active and hover classes/just use :hover and :active

Reply:
we need these for the examples

Reply:
Please include that in notes tab